lib/linguist/vendor.yml

```diff
@@ -276,6 +283,9 @@
 - (^|/)gradle/wrapper/
 
 ## Java ##
+# Java with maven based directory structure
+- /src/main/resources/static/vendor/.*
+- /src/test/resources/static/vendor/.*
```

Comment:
🤔 This should already be caught by these:

https://github.com/github/linguist/blob/2e6c829c3328c5a762182e5b90336839066eceed/lib/linguist/vendor.yml#L89-L93

```ruby
irb(main):001:0> require 'yaml'
=> true
irb(main):002:0> vendored_paths = YAML.load_file(File.expand_path("../lib/linguist/vendor.yml", __FILE__))
=> # ... truncated for brevity ...
irb(main):003:0> VendoredRegexp = Regexp.new(vendored_paths.join('|'))
=> # ... truncated for brevity ...
irb(main):004:0> "/src/main/resources/static/vendor/foo.bar" =~ VendoredRegexp
=> 26
irb(main):005:0> "/src/test/resources/static/vendor/foo.bar" =~ VendoredRegexp
=> 26
irb(main):006:0>
```

Reply:
I was wondering why line 91 didn't catch all those vendored files in my repo when I first pushed the repo. Any ideas?

Reply:
I've just tested again with a clone of your repo, after removing the .gitattributes, and it _does_ catch and thus ignore them:

```
$ git rm .gitattributes 
rm '.gitattributes'
$ git commit -m 'Remove attribs'
[master 7152f16] Remove attribs
 1 file changed, 2 deletions(-)
 delete mode 100644 .gitattributes
$ /bin/ls -la
total 48
drwxr-xr-x  10 lildude  staff   320  6 Mar 13:36 .
drwxr-xr-x   6 lildude  staff   192  5 Mar 14:52 ..
drwxr-xr-x  13 lildude  staff   416  6 Mar 13:36 .git
-rw-r--r--   1 lildude  staff   276 28 Feb 11:38 .gitignore
-rw-r--r--   1 lildude  staff  1317 28 Feb 11:38 build.gradle
drwxr-xr-x   3 lildude  staff    96 28 Feb 11:38 gradle
-rwxr-xr-x   1 lildude  staff  5299 28 Feb 11:38 gradlew
-rw-r--r--   1 lildude  staff  2260 28 Feb 11:38 gradlew.bat
-rw-r--r--   1 lildude  staff    25 28 Feb 11:38 settings.gradle
drwxr-xr-x   4 lildude  staff   128 28 Feb 11:38 src
$ github-linguist . --breakdown
38.73%  CSS
23.43%  JavaScript
22.82%  Java
15.01%  HTML

Java:
src/main/java/com/kft/OmsApplication.java
[... truncated for brevity ...]
src/test/java/com/kft/oms/service/OffenceServiceImplTest.java

CSS:
src/main/resources/static/css/autocomplete.css
src/main/resources/static/css/offence-code-list.css
src/main/resources/static/css/offence-form.css
src/main/resources/static/libs/css/shards-dashboards.1.1.0.css
src/main/resources/static/libs/css/shards.css

JavaScript:
src/main/resources/static/js/layout.js
src/main/resources/static/js/offence-form.js
src/main/resources/static/js/offence-list.js
src/main/resources/static/js/offence-show.js
src/main/resources/static/libs/js/shards.js

HTML:
src/main/resources/templates/association/list.html
src/main/resources/templates/driver/list.html
src/main/resources/templates/layout.html
src/main/resources/templates/offence-code/list.html
src/main/resources/templates/offence/form.html
src/main/resources/templates/offence/list.html
src/main/resources/templates/offence/show.html
src/main/resources/templates/offence/status.html
src/main/resources/templates/offender/offence-list.html
src/main/resources/templates/vehicle/list.html

$
```

As you can see, there's no mention of `vendor` in any of the paths. This is the same analysis as I posted in your original issue.

Your `.gitattributes` changed this because this line:

```
/src/main/resources/static/libs/* linguist-vendored
```

... causes these files from the output above to be marked as vendored:

```
src/main/resources/static/libs/css/shards-dashboards.1.1.0.css
src/main/resources/static/libs/css/shards.css
src/main/resources/static/libs/js/shards.js
```

... and these files are sufficiently large to affect the overall bytes of code percentage for CSS and JS such that they drop below Java:

```console
$ /bin/ls -lh src/main/resources/static/libs/css/shards-dashboards.1.1.0.css \
src/main/resources/static/libs/css/shards.css \
src/main/resources/static/libs/js/shards.js
-rw-r--r--  1 lildude  staff   140K 28 Feb 11:38 src/main/resources/static/libs/css/shards-dashboards.1.1.0.css
-rw-r--r--  1 lildude  staff   111K 28 Feb 11:38 src/main/resources/static/libs/css/shards.css
-rw-r--r--  1 lildude  staff   126K 28 Feb 11:38 src/main/resources/static/libs/js/shards.js
$
```